<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbulEduExerciceWidgetTelecommandeV1</class>
 <widget class="QWidget" name="AbulEduExerciceWidgetTelecommandeV1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>1100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="framePopupQuitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>974</y>
     <width>124</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #cccccc;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="AbulEduFlatBoutonV1" name="btnQuitterRetourMenuPrincipal">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: orange;</string>
      </property>
      <property name="text">
       <string>Fermer</string>
      </property>
      <property name="CouleurTexteSurvol" stdset="0">
       <color>
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </property>
      <property name="CouleurTexteNormale" stdset="0">
       <color>
        <red>255</red>
        <green>165</green>
        <blue>0</blue>
       </color>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="AbulEduFlatBoutonV1" name="btnQuitterAnnuler">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
      <property name="CouleurTexteSurvol" stdset="0">
       <color>
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lbImageFond">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>1100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="AbulEduFlatBoutonV1" name="btnVerifier">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>498</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Vérification</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNiveauEnCours">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>498</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="AbulEduFlatBoutonV1" name="btnQuitter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>740</y>
     <width>498</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sortie</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="AbulEduFlatBoutonV1" name="btnAide">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>620</y>
     <width>498</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Aide</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="AbulEduFlatBoutonV1" name="btnRefaire">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>498</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Début</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="AbulEduFlatBoutonV1" name="btnSuivant">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>498</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Suivant</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="AbulEduFlatBoutonV1" name="btnCorriger">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>680</y>
     <width>498</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Solution</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lblImageTop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblCustom1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>890</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblCustom2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>890</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AbulEduFlatBoutonV1</class>
   <extends>QPushButton</extends>
   <header>abuleduflatboutonv1.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
